skabase/SKALogger/SKALogger.py

```diff
@@ -224,12 +228,15 @@ def SetElementLoggingLevel(self, argin):
         element_logging_device = argin[1][:]
         i = 0
         while i < len(element_logging_level[:]):
-            self.info_stream("Element Logging level : %s, Device : %s",
-                             element_logging_level[i],
-                             element_logging_device[i])
-            dev_proxy = DeviceProxy(element_logging_device[i])
-            dev_proxy.elementLoggingLevel = element_logging_level[i]
-            i += 1
+            try:
+                self.info_stream("Element Logging level : %s, Device : %s",
+                                 element_logging_level[i],
+                                 element_logging_device[i])
+                dev_proxy = DeviceProxy(element_logging_device[i])
+                dev_proxy.elementLoggingLevel = element_logging_level[i]
+                i += 1
```

Comment:
Same comments as above.

Reply:
Added original exception message. Commit id 35e9e42 addresses this.